setup.py

```diff
@@ -417,6 +417,13 @@ def _quote_build_define(argument):
 # TODO(vigneshbabu): Remove this once the poll poller gets fork support.
 DEFINE_MACROS += (('GRPC_DO_NOT_INSTANTIATE_POSIX_POLLER', 1),)
 
+# Fix for Cython build issue in aarch64.
```

Comment:
Please add an issue and a TODO to figure out what's going on in the aarch64 build so that we can supports gcc + Bazel.